src/T2.java

```diff
@@ -0,0 +1,9 @@
+public class T2 {
+    public static void main (String[] args) {
+        System.out.println("OK");
+    }
+  
+    public int newMethod(int a, int b, int c, int d, int e) {
```

Comment:
[Code Smell Detector Automated Comment]
This method contains more than the recommended number of arguments.